pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -55,6 +55,10 @@ mkDerivation rec {
     sqlite
   ];
 
+  qtWrapperArgs = [
+    "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libsecret ]}"
```

Comment:
```suggestion
    "--prefix LD_LIBRARY_PATH : ${lib.getOutput "lib" libsecret}/lib"
```

Reply:
@zseri, can you explain what these changes do?

Reply:
This basically just removed the indirection through `makeLibraryPath`, as it is only called with one argument, and thus the indirection (mapping + string concat) is not needed. But it would only make sense if it is not expected that additional paths might be added in the near future.

Reply:
I understand, well I don't know if there will be a future need, so I would leave that point to @caugner.

Thanks for the explanation: P

Reply:
the `makeLibraryPath` looks right to me